casper/src/test/scala/coop/rchain/casper/util/comm/TransportLayerTestImpl.scala

```diff
@@ -34,15 +35,16 @@ class TransportLayerTestImpl[F[_]: Monad](
   def broadcast(peers: Seq[PeerNode], msg: Protocol): F[Seq[CommErr[Unit]]] =
     peers.toList.traverse(send(_, msg)).map(_.toSeq)
 
-  // FIX-ME handleStreamed not handled (pun)
   def receive(
       dispatch: Protocol => F[CommunicationResponse],
       handleStreamed: Blob => F[Unit]
   ): F[Unit] =
     TransportLayerTestImpl.handleQueue(dispatch, msgQueues(identity))
 
-  // FIX-ME
-  def stream(peers: Seq[PeerNode], blob: Blob): F[Unit] = ???
+  def stream(peers: Seq[PeerNode], blob: Blob): F[Unit] =
+    peers.toList
+      .traverse(peer => send(peer, protocol(blob.sender).withPacket(blob.packet)))
+      .map(_.toSeq)
```

Comment:
What is the point of this `map` since the return type is `F[Unit]`?

Or better yet, why not 
```scala
def stream(peers: Seq[PeerNode], blob: Blob): F[Unit] =
  broadcast(peers, protocol(blob.sender).withPacket(blob.packet)).void
```